### ISSUE TITLE
Build phase is using wrong perl (0.001-TRIAL, 0.002-TRIAL)

### ISSUE THREAD

**Issue by eserte** [jddurand/re-engine-GNU#1]
It seems that during build phase the wrong perl is used --- the system perl instead of the perl used for running Makefile.PL or Build.PL. This leads to errors like this:

```
...
perl config_autoconf.pl
Can't locate Config/AutoConf.pm in @INC (you may need to install the Config::AutoConf module) (@INC contains: /usr/local/lib/perl5/site_perl/mach/5.18 /usr/local/lib/perl5/site_perl /usr/local/lib/perl5/5.18/mach /usr/local/lib/perl5/5.18 /usr/local/lib/perl5/site_perl/5.18 /usr/local/lib/perl5/site_perl/5.18/mach .) at config_autoconf.pl line 4.
BEGIN failed--compilation aborted at config_autoconf.pl line 4 (#1)
    (F) You said to do (or require, or use) a file that couldn't be found.
    Perl looks for the file in all the locations mentioned in @INC, unless
    the file name included the full path to the file.  Perhaps you need
    to set the PERL5LIB or PERL5OPT environment variable to say where the
    extra library is, or maybe the script needs to add the library name
    to @INC.  Or maybe you just misspelled the name of the file.  See
    "require" in perlfunc and lib.

Uncaught exception from user code:
        Can't locate Config/AutoConf.pm in @INC (you may need to install the Config::AutoConf module) (@INC contains: /usr/local/lib/perl5/site_perl/mach/5.18 /usr/local/lib/perl5/site_perl /usr/local/lib/perl5/5.18/mach /usr/local/lib/perl5/5.18 /usr/local/lib/perl5/site_perl/5.18 /usr/local/lib/perl5/site_perl/5.18/mach .) at config_autoconf.pl line 4.
        BEGIN failed--compilation aborted at config_autoconf.pl line 4.
*** [config_autoconf.h] Error code 2

Stop in /usr/home/eserte/.cpan/build/re-engine-GNU-0.002-_C7BIh.
```

**Comment by jddurand** [comment]
Oh... many thanks.
I was trying to understand why it fails while it is perfectly ok in my perlbrews's...
probably $^X instead of just perl in the dist.ini then
Thanks, again.

**Comment by jddurand** [comment]
Ah..; no in the context of a Makefile, it should be $(PERLRUN) familly

**Comment by jddurand** [comment]
Fixed with f70a35c32fdbb8cd0e6477b60b4c5304c42a394e

**Comment by eserte** [comment]
0.004-TRIAL looks good.
